src/@types/topic.ts

```diff
@@ -42,20 +42,17 @@ import { type IMessageRendering } from "./extensible_events.ts";
 /**
  * The event type for an m.topic event (in content)
  */
-export const M_TOPIC = new UnstableValue("m.topic", "org.matrix.msc3765.topic");
+export const M_TOPIC = new NamespacedValue("m.topic");
```

Comment:
Shouldn't this still reference the unstable value so consumers can continue to render existing topics as they did before rather than break all existing rooms?

Reply:
I had some [trouble](https://github.com/matrix-org/matrix-js-sdk/pull/4767#discussion_r2018054292) getting it to work correctly. Maybe you have an idea how to do this properly?